packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.html

```diff
@@ -115,4 +115,26 @@
       </template>
     </div>
   </template>
+  <template lwc:if={shouldDisplayNoAnswerMessage}>
+      <div data-testid="generated-answer__card-no-answer" class="slds-box">
+        <div class="generated-answer__card-header slds-grid">
+          <span
+            data-testid="generated-answer__badge"
+            class="generated-answer__badge slds-var-p-around_x-small slds-text-title_bold"
+            >{labels.generatedAnswerForYou}</span
+          >
+        </div>
+          <div
+            data-cy="generated-answer__content"
+            class="generated-answer__content slds-var-m-top_medium"
+          >
+            <div data-testid="generated-answer__no-answer" class={generatedAnswerClass}>
```

Comment:
so the id makes a bit more sense.
```suggestion
            <div data-testid="generated-answer__no-answer-message" class={generatedAnswerClass}>
```